gammapy/detect/cwt.py

```diff
@@ -429,7 +429,7 @@ class CWTData(object):
     def __init__(self, counts, background, n_scale):
         self._counts = np.array(counts.data, dtype=float)
         self._background = np.array(background.data, dtype=float)
-        self._wcs = counts.wcs
+        self._wcs = counts.geom
```

Comment:
Can you call this `_geom` please, instead of `wcs`. Note that there is a `geom.wcs`, so if we don't distinguish between geom and wcs for Astropy WCS objects, it will be very confusing.

Also note that @adonath recently added a `map.copy` and `geom.copy` methods for convience.
You could use those here, especially importing and calling `copy.deepcopy` shouldn't be needed any more.